pipelines/alert/rocketchat.yaml

```diff
@@ -0,0 +1,160 @@
+---
+# Description
+#   The Sensu RocketChat Handler is a Sensu Event Handler that sends alerts to 
+#   RocketChat. 
+#
+# Instructions
+#   1. Add "rocketchat" to the "alert" handler set.
+#
+#      To create the "alert" handler set, please run:
+#
+#      $ sensuctl handler create alert --type set --handlers rocketchat
+#
+#      If you already have a handler set called "alert", please run:
+#
+#      $ sensuctl edit handler alert
+#
+#      NOTE: the sensuctl edit command will open your default editor; please
+#      modify the "handlers" attribute by adding "rocketchat".
+#
+#   2. Configure the following secrets using your preferred Sensu Go Secrets
+#      provider:
+#
+#      - rocketchat_user
+#      - rocketchat_password
+#      - rocketchat_channel
+#      - rocketchat_url
+#
+#      NOTE: example secret resources using the "env" provider are included in
+#      this template; to add your secrets to the Sensu Backend, please modify
+#      /etc/default/sensu-backend or /etc/sysconfig/sensu-backend and restart
+#      the sensu-backend service.
+#
+#   3. More configuration options are available for this plugin; please see the
+#      plugin documentation for more details.
+#
+# Documentation
+#   - https://github.com/sensu/sensu-rocketchat-handler
+#   - https://docs.sensu.io/sensu-go/latest/reference/secrets/
+#
+# Contributors
+#   The following individuals have contributed to this configuration template:
+#   - @calebhailey
+type: Handler
+api_version: core/v2
+metadata:
+  name: rocketchat
+spec:
+  type: pipe
+  command: >-
+    sensu-rocketchat-handler --channel ${ROCKETCHAT_CHANNEL}i --url ${ROCKETCHAT_URL}
```

Comment:
Is there a typo here (a superfluous `i`)? `--channel ${ROCKETCHAT_CHANNEL}i`

Reply:
vim insert mode strikes again!!!!!